app/src/main/kotlin/com/wire/android/ui/WireActivity.kt

```diff
@@ -285,4 +284,27 @@ class WireActivity : AppCompatActivity() {
         super.onPause()
         proximitySensorManager.unRegisterListener()
     }
+
+    override fun onSaveInstanceState(outState: Bundle) {
+        outState.putBoolean(HANDLED_DEEPLINK_FLAG, true)
```

Comment:
I'm not sure, what if we started the app, got a deep link, handled it so it's set to true and then got another deep link?

Reply:
in that case deeplink intent goes into `onNewIntent` fun, which ignores that flag